web/app/scripts/resources/query-builder-service.js

```diff
@@ -12,10 +12,14 @@
     function QueryBuilder($q, FilterState, RecordState, Records) {
         var svc = this;
         svc.djangoQuery = djangoQuery;
-        svc.unfilteredDjangoQuery = function(offset) { return djangoQuery(offset, false); };
+        svc.unfilteredDjangoQuery = function(extraParams, offset) {
+            return djangoQuery(extraParams, offset, false);
+        };
         svc.windshaftQuery = windshaftQuery;
-        svc.unfilteredWindshaftQuery = function() { return windshaftQuery(false); };
+        svc.unfilteredWindshaftQuery = function() { return windshaftQuery({}, false); };
+        // KEEP THESE AVAILABLE FOR TESTING
         svc.assembleParams = assembleParams;
+        svc.assembleJsonFilterParams = assembleJsonFilterParams;
```

Comment:
This could use some cleanup to match our Angular style guidelines more closely: https://github.com/azavea/azavea-coding-guidelines/blob/master/javascript/angularjs.md#structure-of-angular-components